src/pages/List/index.js

```diff
@@ -1,14 +1,26 @@
 import React from 'react';
-import Header from '../../components/title';
 import Navigation from '../../components/routing/Navigation';
 import { ShowProducts } from '../../components/show-products';
 
+import '../../components/show-products/styles.css';
+import ContentContainer from '../../components/content-container';
+
+import Image from '../shopping-add-image.svg';
+
+import '../pages-styles.css';
+
 export const ListPage = () => {
   return (
-    <>
-      <Header className="page-header">List</Header>
-      <ShowProducts />
-      <Navigation />
-    </>
+    <main className="page-container">
+      <div className="image-container">
+        <img src={Image} alt="Shopping" width="80%" />
```

Comment:
Interesting! How you are passing an SVG to `<img />`. 

I have a small question: is it possible to move `width=80%` to a CSS class?

Reply:
Thanks!

I made the suggested fixes!